weaveworks/weave-gitops-enterprise
# fix button spaces

Closes [#3891](https://github.com/weaveworks/weave-gitops/issues/3891)


**What changed?**

This PR related to [#3985](https://github.com/weaveworks/weave-gitops/pull/3985) PR in `weave-gitops` to fix [#3891](https://github.com/weaveworks/weave-gitops/issues/3891) issue  to update `ActionWrapper` for buttons with a gap to fix spacing between them.


**How did you validate the change?**

 
   1. Clone `adjust-spacing` branch from `weave-gitops` 
   2. navigate to **/ui-cra** folder in `EE`
   3. run `make core-ui && make core-lib`
   4. run `WEGO_EE_PROXY_HOST=https://demo-01.wge.dev.weave.works//  yarn start`
